src/stories/Forms/Fieldset/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.fieldset`
+  display: block;
+  min-width: 0;
+  max-width: 400px;
+  margin: 0 0 var(--spacing-32) 0;
+  padding: 0.01em 0 0 0;
```

Comment:
What is this cryptic 0.01em spacing? Sound like a very small padding?

Reply:
Will write a comment, but it's for resetting fieldset. Some browsers ignore `padding: 0`.

Reply:
I learn something new every day 💡